index.html

```diff
@@ -1022,7 +1022,7 @@ <h3>Errors</h3>
  where <code>1234</code> is not the <a>session id</a> of a <a>current session</a>
  would return an <a>HTTP response</a> with the status 404 and a body of the form:
 
- <pre><code>{
+ <pre class=nohighlight><code>{
```

Comment:
Let try this out... you shouldn’t neeed to do this.

Reply:
Did something change in the Puppeteer dependency?

I dropped the commit adding the `nohighlight` class, and it appears to pass in CI, so thanks!

Reply:
> Did something change in the Puppeteer dependency?

Lastest version broke something in Web Workers... highlighting is done using a web worker.